Components/InputTypeRadio/index.js

```diff
@@ -0,0 +1,39 @@
+import styled from "styled-components";
+
+export default function InputTypeRadio({
+  nameString,
+  valueString,
+  onChange,
+  idString,
+  labelString,
+}) {
+  function checked() {
+    nameString === valueString;
+  }
+
+  return (
+    <>
+      <RadioButton>
+        <input
+          type="radio"
+          name={nameString}
+          value={valueString}
+          id={idString}
+          onChange={onChange}
+          checked={checked()}
+        />
+        <Label htmlFor={idString}>{labelString}</Label>
+      </RadioButton>
+    </>
```

Comment:
No need for React Fragment when you only return one element. 

```suggestion
      <RadioButton>
        <input
          type="radio"
          name={nameString}
          value={valueString}
          id={idString}
          onChange={onChange}
          checked={checked()}
        />
        <Label htmlFor={idString}>{labelString}</Label>
      </RadioButton>
```

Reply:
Done.